app/src/main/java/com/example/lamnn/todo_mvp/task/TasksPresenter.java

```diff
@@ -0,0 +1,48 @@
+package com.example.lamnn.todo_mvp.task;
+
+import com.example.lamnn.todo_mvp.data.model.Task;
+import com.example.lamnn.todo_mvp.data.repository.TasksRepository;
+import com.example.lamnn.todo_mvp.data.resource.Callback;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TasksPresenter implements TasksContract.Presenter {
+
+    TasksContract.View mView;
```

Comment:
missing access modifier 
```suggestion
     private TasksContract.View mView;
```